packages/build/tests/core/tests.js

```diff
@@ -37,7 +37,7 @@ const getNodeBinary = async function (nodeVersion, retries = 1) {
   }
 }
 
-const mGetNode = moize(getNodeBinary, { isPromise: true, maxSize: 1e3 })
+const mGetNode = memoize(getNodeBinary, { async: true, maxSize: 1e3 })
```

Comment:
👍🏼 Comparing https://github.com/planttheidea/moize and https://github.com/planttheidea/micro-memoize, these seem to be equivalent